src/Components/CategoryCard.jsx

```diff
@@ -1,18 +1,30 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
-export const CategoryCard = ({ data }) => {
+export const CategoryCard = ({ data, gender, itemCategory }) => {
+  //Url için gender propunu lowercase'e dönüştürme.
+  const genderUrl = gender.toLowerCase();
+  //Apiden Gelen -> "code": "k:elbise" verisini splitleyip "elbise" kısmını kullanma.
+  const urlSplit = itemCategory.split(":");
+  const categoryUrl = urlSplit[1];
   return (
-    <div className="max-w-[200px] md:w-[333px] md:mx-auto">
-      {/* Linkler nereye gidecek? */}
-      <Link>
+    <div className="md:mx-auto md:w-full ">
+      {/* Kategori linki nasıl olmalı? , nereye gidecekler? */}
+      <Link
+        to={`/shopping/:${genderUrl}/:${categoryUrl}`}
```

Comment:
burada syntax hatalı iki nokta sadece okurken kullanılıyor. https://v5.reactrouter.com/web/api/Link 

Bu örnekte `:id` dediği dinamik değişkeni okumak için kullanılıyor. https://v5.reactrouter.com/web/example/url-params Link'i ise, örnekteki satır 24'deki gibi SABİT url'ler vererek kullanıyoruz. 

`to={`/shopping/${genderUrl}/${categoryUrl}`}` olmalı